x-pack/plugins/reporting/public/lib/reporting_api_client/reporting_api_client.ts

```diff
@@ -134,6 +150,16 @@ export class ReportingAPIClient implements IReportingAPI {
     return new Job(report);
   }
 
+  public async getPayload(jobId: string) {
+    const payload: ReportSource['payload'] = await this.http.get(
+      `${API_LIST_URL}/payload/${jobId}`,
+      {
+        asSystemRequest: true,
```

Comment:
If the method is called as a result of a user action, such as clicking a button, then it should not have this flag. It means Kibana doesn't treat the call as a user action and the expiration of their login session doesn't get extended. 

In this API Client, the only method that should use a system request is the `list` and `total` method. 

I will approve this PR as-is since this was a pre-existing minor bug.